examples/index.tsx

```diff
@@ -40,6 +40,9 @@ ReactDOM.render((
       <ul className="nav">
         <li><NavLink to="/">All Features Example</NavLink></li>
```

Comment:
We missed this in the previous PR, but we should add the `exact` prop here so we don't always highlight this link:
```suggestion
        <li><NavLink to="/" exact>All Features Example</NavLink></li>
```

Reply:
will create a new PR to add this.